crates/ra_ide_api/Cargo.toml

```diff
@@ -4,6 +4,9 @@ name = "ra_ide_api"
 version = "0.1.0"
 authors = ["rust-analyzer developers"]
 
+[features]
+wasm = []
```

Comment:
Is there a reason this is a feature? You should be able to just check for wasm via a normal cfg based on the target?!

Reply:
https://github.com/rust-analyzer/rust-analyzer/pull/1746#discussion_r319906852

Not sure which is better, but, with a feature, we can check that wasm build actually builds without needing to compile to wasm

Reply:
Ah, looks like you already had a discussion about this further up.